internal/kokoro/test.sh

```diff
@@ -56,3 +56,10 @@ if [[ $KOKORO_JOB_NAME == *"continuous"* ]]; then
 else
     go test -race -v -short ./... 2>&1 | tee $KOKORO_ARTIFACTS_DIR/$KOKORO_GERRIT_CHANGE_NUMBER.txt
 fi
+
+if [[ $KOKORO_BUILD_ARTIFACTS_SUBDIR = *"continuous"* ]]; then
```

Comment:
Do we only want to run flakybot on `continuous`, or should we always run it in this file, meaning for `go test -race -v -short ...` as well?

Reply:
just continuous.